zeebe/dynamic-config/src/test/java/io/camunda/zeebe/dynamic/config/gossip/ClusterConfigurationGossiperTest.java

```diff
@@ -82,16 +82,11 @@ void shouldPropagateTopologyUpdate(final ClusterConfigurationGossiperConfig conf
 
   private static Stream<Arguments> provideConfig() {
     return Stream.of(
-        Arguments.of(
-            Named.of(
-                "by gossip", // Disable sync
-                new ClusterConfigurationGossiperConfig(
-                    false, Duration.ofMinutes(10), Duration.ofSeconds(1), 2))),
         Arguments.of(
             Named.of(
                 "by sync", // Set gossipFanout to 0
                 new ClusterConfigurationGossiperConfig(
-                    true, Duration.ofMillis(100), Duration.ofSeconds(1), 0))));
+                    Duration.ofMillis(100), Duration.ofSeconds(1), 0))));
```

Comment:
:wrench:  This test doesn't need to be parameterized any more since there is only once config that is being tested.